apps/web/src/pages/Applications/ApplicationSelfDeclarationPage/SelfDeclaration/SelfDeclarationForm.test.tsx

```diff
@@ -108,30 +115,30 @@ describe("SelfDeclarationForm", () => {
     ).toBeInTheDocument();
   });
 
-  it("should display status field if Indigenous and First Nations", async () => {
-    await act(async () => {
-      renderSelfDeclarationForm();
-    });
+  // it("should display status field if Indigenous and First Nations", async () => {
```

Comment:
Oops removed with [ec63948](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7140/commits/ec639483cb5c8ce71ef6506116dda66dbaf25814) since it doesn't make sense anymore.